README.md

```diff
@@ -1,5 +1,5 @@
-# 🤖 Online Chess Cheater Bot
-[lichess.org](https://lichess.org/) and [chess.com](https://chess.com) are the two most popular online chess platforms. This application provides the user with the top engine move and position evaluation. Once a live online game is started, it automatically pulls the played moves from the webpage's source and builds it own chess board. A local stockfish engine then analyzes the position and offers the best move to the user.
+# 🤖 Online Chess Bot
```

Comment:
let's not hide the real (evil) purpose of this project :()

Lichess has its own "Bot" players but here an actual human logs in, so...